nautobot/extras/tests/test_customfields.py

```diff
@@ -1458,7 +1460,7 @@ def setUpTestData(cls):
 
     def test_filter_integer(self):
```

Comment:
Would be good to change all of these to `assertQuerysetEqual()`, e.g.:

```python
self.assertQuerysetEqual(self.filterset({"cf_cf1": 100}, self.queryset).qs, self.queryset.filter(_custom_field_data__cf1=100))
self.assertQuerysetEqual(self.filterset({"cf_cf1__n": [100]}, self.queryset).qs, self.queryset.exclude(_custom_field_data__cf1=100))
```

Not blocking.